src/rm.js

```diff
@@ -48,7 +48,25 @@ function rmdirSyncRecursive(dir, force) {
 
   var result;
   try {
-    result = fs.rmdirSync(dir);
+    // Retry on windows, sometimes it takes a little time before all the files in the directory are gone
+    var start = Date.now();
+    while (true) {
+      try {
+        result = fs.rmdirSync(dir);
+        if (fs.existsSync(dir)) throw { code: "EEXIST" }
```

Comment:
Let me know what you want to do about it and I'll change it. 

@Schoonology Regarding 1 and 2: I'm not sure how you would concatenate a good message with an actual `Error` without creating a new `Error` class. Could you clarify?

@arturadib The code flow goes to the outer `catch` which does use `common.error()`. That throw there is just to give it a chance to retry. It's handled by the inner `catch` in the `if`. If the error is re-thrown, the outer `catch` handles it normally.